Include/cpython/pyhash.h

```diff
@@ -15,9 +15,14 @@
 #  define _PyHASH_BITS 31
 #endif
 
+#define PyHASH_BITS _PyHASH_BITS
```

Comment:
I would prefer to do the opposite: keep the old name as a deprecated alias, but define the public name to the value: 3 lines above. Same remark for other constants.

Reply:
Ok, I did that.  Deprecated aliases added below in a separate block.